documents/Final Report/outlook.tex

```diff
@@ -4,10 +4,6 @@ \chapter{Outlook}
 \section {Product}
 Our current agent has no way to find out how a certain indicator can be improved, we had to hardcode the names of the indicators. This could be a big improvement to make the agent a lot more dynamic. Changes in the connector and possibly in the SDK are needed for this, but it would make programming an agent a lot more convenient.
 
-Another improvement would be to change the way land works, our agent has to work with multipolygons currently which makes most computations impossible. Because of this a lot of functionality is moved to the connector. Having to make a function call to the environment for every calculation with land does not work very well for an agent that has to do city planning. If we get a request to buy a piece of land for example, we have to know if there are buildings on this land, if it has a good shape to build buildings and where the land is exactly, to make a good decision about this. But all of this information our agent can not figure out by itself. A grid of the landscape where every building is build within cells would be a solution for this, making it a lot easier to see collisions and locations. This would require a lot of changes to the connector though, but would enable our agent with a lot more possibilities.
+Another improvement would be to change the way land works, our agent has to work with multipolygons currently which makes most computations impossible. Because of this a lot of functionality is moved to the connector. Having to make a function call to the environment for every calculation with land does not work very well for an agent that has to do city planning. For example, if a stakeholder wants to sell us a piece of land, our agent should make a good decision about this. To make a good decision it needs to know where the land is, what buildings are on it and if it is suitable for building. But all of this information our agent can not figure out by itself. A grid of the landscape where every building is build within cells would be a solution for this, making it a lot easier to see collisions and locations. This would require a lot of changes to the connector and api though, but would enable our agent with a lot more possibilities.
```

Comment:
"our agent has to work with multipolygons currently" maybe change this to "our agent currently has to work with multipolygons" this makes it nicer to read